org-drill-table.el

```diff
@@ -154,7 +155,11 @@ and the row value."
       (if (zerop idx) (org-insert-subheading nil) (org-insert-heading))
       (insert header)
       (newline)
-      (insert value))))
+      (org-indent-line)
+      (insert value)
+      (org-indent-line)
```

Comment:
Hi, I'm happy to make the change but I'm not clear on what you mean by that. Can you please clarify?